src/GeometryOps.jl

```diff
@@ -6,16 +6,21 @@ using GeoInterface
 using GeometryBasics
 import Tables
 using LinearAlgebra, Statistics
-import GeometryBasics.StaticArrays
 import Base.@kwdef
 
 using GeoInterface.Extents: Extents
 
 const GI = GeoInterface
 const GB = GeometryBasics
+const SA = GeometryBasics.StaticArrays
 
 const TuplePoint{T} = Tuple{T, T} where T <: AbstractFloat
-const Edge{T} = Tuple{TuplePoint{T},TuplePoint{T}} where T
+const TupleEdge{T} = Tuple{TuplePoint{T},TuplePoint{T}} where T
+
+const SVPoint{T} = GI.Point{false, false, SA.SVector{2, T}, Nothing} where T <: AbstractFloat
```

Comment:
I'm not sure how to enable `SVPoint{Z, M, T}` - tried to do it the obvious way with:
```julia
const SVPoint{Z, M, T, CRS} = GI.Point{Z, M, SA.SVector{2+Z+M}, T, CRS}
```
but that complains about Z and M being type variables.  Any thoughts on how we could accomplish this would be great :D

Reply:
Yeah... this is also going to be needed for #97  so that we can actually store the extents...

I also don't know how to do it 😆 which is why I hard-coded for now

Reply:
Also, the functions that use it only currently work for 2D so it isn't the end of the world right now, but definitely a needed improvement.